imports/plugins/included/shipping/server/methods.js

```diff
@@ -1,19 +1,26 @@
 import { Shipping } from "/lib/collections";
 import { Reaction } from "/server/api";
 
-Meteor.methods({
-  /*
+export const methods = {
+  /**
```

Comment:
Is this the standard on how we are doing this now? If so I like it except I have been doing one thing differently in that I have kept the Role/Permission checks in the Meteor method but not in the exported method so that they can be called in code (e.g. hooks) without having a logged in user. I don't feel like we need to check permissions if they are being called from server-side code. Not sure what your thoughts are.